src/main/webapp/resources/js/pages/projects/ncbi/create/CreateNcbiSampleDetails.tsx

```diff
@@ -0,0 +1,231 @@
+import { SwapOutlined, SwapRightOutlined } from "@ant-design/icons";
+import {
+  Avatar,
+  Cascader,
+  Checkbox,
+  Col,
+  Form,
+  FormInstance,
+  Input,
+  List,
+  Row,
+  Select,
+  Space,
+} from "antd";
+import React from "react";
+import { useLoaderData } from "react-router-dom";
+import {
+  DefaultModifiableField,
+  LoaderValues,
+  SampleRecord,
+} from "./CreateNcbiExport";
+
+const rules = [
+  {
+    required: true,
+    message: i18n("CreateNcbiExport.required"),
+  },
+];
+
```

Comment:
Component comment

Reply:
Updated in b3e0ce2037245a6a1aa531b6f09a9cba83e73999